demos/dotnet/dotnet/dotnet.cs

```diff
@@ -712,13 +735,6 @@ public static bool RestorePackagesAction(ProjectProperties properties, Log log)
             Console.WriteLine(e);
             return false;
         }
-        finally
-        {
-            if (process != null)
-            {
-                process.Close();
```

Comment:
Yes it is a good idea to call Dispose. You should just add a using statement around the Process usage.